octavia-cli/integration_tests/test_generate/expected_rendered_yaml/destination_s3/input_spec.yaml

```diff
@@ -180,9 +180,9 @@ spec:
                         default: 3
                         minimum: -5
                         maximum: 22
-                      include_hash:
-                        title: "Include hash"
-                        description: "If true, include a hash with each data block."
+                      include_checksum:
```

Comment:
@alafanechere are you chancing to checksum?

Reply:
Here the original field name for this S3 destination connector was `include_checksum`. I mistakenly renamed it to `include_hash` while doing a brutal search and replace it when renaming other code occurrences of `checksum` to `hash`.